agent/api/serviceconnect/service_connect_validator.go

```diff
@@ -171,7 +172,7 @@ func validateIngressConfig(scIngressConfigList []IngressConfigEntry, taskNetwork
 	}
 
 	switch taskNetworkMode {
-	case BridgeNetworkMode, AWSVPCNetworkMode:
+	case BridgeNetworkMode, AWSVPCNetworkMode, DefaultNetworkMode:
```

Comment:
Do we need this switch at all if we are going to validate for all scenarios?
If yes, could we instead set the network mode when it is not set?

Reply:
Yeah, I found out that the _real_ problem here is that we are validating before detecting the network mode. I've updated the code to accommodate that.

Reply:
So effectively this means networkmode will always be set by the time it hits this use case.